vng_api_common/audittrails/constants.py

```diff
@@ -0,0 +1,11 @@
+from django.utils.translation import ugettext_lazy as _
+
+from djchoices import ChoiceItem, DjangoChoices
+
+
+class AuditTrailAction(DjangoChoices):
```

Comment:
Dit mag van mij generiek worden (`CommonResourceAction`) en in een niveau hoger worden verplaatst (in `vng_api_common.constants`).